caravel/templates/listings/fullpage.html

```diff
@@ -47,6 +47,19 @@ <h3 class="panel-title">Manage Listing</h3>
           <h3 class="panel-title">Contact Seller</h3>
         </div>
         <div class="panel-body">
+        {% if session["email"] in listing.buyers %}
+          <p class="alert alert-success">
+            You have successfully expressed interest in this listing.</p>
+        {% else %}
+          {% if listing.buyers %}
+          <p class="alert alert-info">
+            {% if listing.buyers|length != 1 %}
+              This listing has {{ listing.buyers|length }} inquries.
+            {% else %}
+              This listing has 1 inqury.
```

Comment:
Oh so if there is not listing.buyer, then it won't display anything? 

I guess my point was, do we want the default to an icon for 0 buyers?